src/main/scala/org/broadinstitute/dsde/firecloud/service/SubmissionService.scala

```diff
@@ -22,20 +18,5 @@ trait SubmissionService extends HttpService with PerRequestCreator with FireClou
   lazy val log = LoggerFactory.getLogger(getClass)
 
   def postAndGetRoutes: Route =
-    pathPrefix("workspaces" / Segment / Segment / "submissions") { (workspaceNamespace, workspaceName) =>
-      val listSubmissionsPath = FireCloudConfig.Rawls.submissionsUrl(workspaceNamespace, workspaceName)
-      pathEnd {
-        passthrough(listSubmissionsPath, HttpMethods.GET, HttpMethods.POST)
-      } ~
-      pathPrefix(Segment) { submissionId =>
-        pathEnd {
-          passthrough(FireCloudConfig.Rawls.
-            submissionByIdUrl(workspaceNamespace, workspaceName, submissionId), HttpMethods.GET, HttpMethods.DELETE)
-        } ~
-        path("workflows" / Segment / "outputs") { workflowId =>
-          passthrough(FireCloudConfig.Rawls.
-            workflowOutputsByIdUrl(workspaceNamespace, workspaceName, submissionId, workflowId), HttpMethods.GET)
-        }
-      }
-    }
+    passthroughAllPaths("workspaces", FireCloudConfig.Rawls.baseUrl + "/workspaces")
```

Comment:
I like the simplification, but this conflicts with routes in WorkspaceService. Admittedly, FireCloudServiceActor checks WorkspaceService first, so it shouldn't conflict in *practice*, but it feels wrong.